signalling/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name="webrtcsink-signalling"
+version = "0.1.0"
+edition = "2018"
+authors = ["Mathieu Duponchelle <mathieu@centricular.com>"]
+license = "MIT"
+description = "GStreamer WebRTC sink signalling server"
+repository = "https://github.com/centricular/webrtcsink/"
+
+[dependencies]
+anyhow = "1"
+async-std = { version = "1", features = ["unstable"] }
+async-tungstenite = { version = "0.16", features = ["async-std-runtime", "async-native-tls"] }
+serde = "1"
+serde_derive = "1"
+serde_json = "1"
+clap = { version = "3", features = ["derive"] }
+tracing = { version = "0.1", features = ["log"] }
+tracing-subscriber = { version = "0.3", features = ["registry", "env-filter"] }
+tracing-log = "0.1"
+futures = "0.3"
+uuid = { version = "0.8", features = ["v4"] }
```

Comment:
```suggestion
uuid = { version = "0.8", features = ["v4", "serde"] }
```

or not?

Reply:
No, peer ids are just required to be String